Runtime/Association/PlatformDeviceAssociation.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Zinnia.Association
+{
+    using Malimbe.PropertySerializationAttribute;
+    using Malimbe.XmlDocumentationAttribute;
+    using System.Text.RegularExpressions;
+    using UnityEngine;
+    using UnityEngine.XR;
+
+    /// <summary>
+    /// Holds <see cref="GameObject"/>s to (de)activate based on the current platform and loaded XR device type.
+    /// </summary>
+    public class PlatformDeviceAssociation : GameObjectsAssociation
+    {
+        /// <summary>
+        /// A regular expression to match the name of the current <see cref="RuntimePlatform"/>.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public string PlatformPattern { get; set; }
```

Comment:
`Application.platform` is using a known type, should this perhaps be a multi-select thing for picking enum values? Or is the issue making that work with the Unity inspector perhaps, and thus regex provides the same wanted outcome?

Reply:
A regular expression will also allow you to fuzzy check the platform, or do multiple platform checks, e.g. Android or Standalone